python-package/mlbox/encoding/categorical_encoder.py

```diff
@@ -429,44 +486,68 @@ def transform(self, df):
                                 d[unknown_level] = new_enc
                                 self.__Enc[col] = d
 
-
-                    if(len(self.__Lnum)==0):
-                        return pd.concat([pd.DataFrame(df[col].apply(lambda x: self.__weights[i][int(self.__Enc[col][x]),:] if int(self.__Enc[col][x])<np.shape(self.__weights[i])[0] else np.mean(self.__weights[i],axis=0)).tolist(),columns=[col+"_emb"+str(k+1) for k in range(self.__K[col])], index = df.index)
-                                            for i, col in enumerate(self.__Lcat)],axis=1)
+                    if (len(self.__Lnum) == 0):
+                        # TODO: Find a way to shorten here below
+                        return pd.concat(
+                            [pd.DataFrame(
+                                df[col].apply(lambda x: self.__weights[i][int(self.__Enc[col][x]), :] if int(self.__Enc[col][x]) < np.shape(self.__weights[i])[0] else np.mean(self.__weights[i], axis=0)).tolist(),
```

Comment:
@NCherel @AxeldeRomblay 

Any ideas on how to break the line so that it has less than 80 characters?
Couldn't manage to do it because of the `if ... else ...` statement in the list comprehension 🤕